### PR TITLE
machines: Don't show empty state pattern to non-privileged users

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: pkg/machines/app.jsx [hunk 1]
```diff
@@ -108,7 +117,8 @@ class App extends React.Component {
                 vms={vms} />
         );
 
-        if (systemInfo.libvirtService.activeState !== 'running') {
+        // Show libvirtSlate component if libvirtd is not running only to users that are allowed to start the service.
+        if (systemInfo.libvirtService.activeState !== 'running' && (this.state.allowed == undefined || this.state.allowed)) {
```
[the remark this answers is not in the document]
Comment (KKoukiou) [reply pkg/machines/app.jsx]: I actually tried it, and it shows the loading timer for a second, I guess that fine.
If the permission took too much time to initialize we would indeed see the 'service start page', but I don't see any reason why that would happen.

[the remark this answers is not in the document]
Comment (martinpitt) [reply pkg/machines/app.jsx]: Ack, thanks. It's only a very minor thing anyway, and we can fine-tune this later.

**Comment by martinpitt** [comment]
Marking priority for the pyflakes fix part (that should have been separate, but let's not worry about that for now, just land this ASAP)

**Comment by martinpitt** [comment]
New [failure](https://209.132.184.41:8493/logs/pull-11928-20190527-124747-268ceefb-verify-rhel-7-6-distropkg/log.html#96):

   testlib.Error: cannot reauthorize identity(s): unix-user:admin

**Comment by martinpitt** [comment]
[This](https://209.132.184.41:8493/logs/pull-11928-20190528-154444-b5c9e8f8-selenium-edge/log.html#50) and [this](https://209.132.184.41:8493/logs/pull-11928-20190528-160101-b5c9e8f8-verify-debian-stable/log.html#77) failure are related, retrying.

**Comment by martinpitt** [comment]
[same debian-stable](https://209.132.184.41:8493/logs/pull-11928-20190528-175050-b5c9e8f8-verify-debian-stable/log.html#99) failure again ("the cockpit package is not installed"), huh?

**Comment by martinpitt** [comment]
Sorry, this seems genuinely broken, third failure like that in a row.

**Comment by KKoukiou** [comment]
@martinpitt  sorry, I wanted to add the '--shell' to debian-stable only, force pushed to conditionally add this argument to useradd.

**Comment by martinpitt** [comment]
It looks like [this failure](https://209.132.184.41:8493/logs/pull-11928-20190529-034646-9ba1a677-verify-debian-testing/log.html#80) is related. It also happens on [fedora-30](https://209.132.184.41:8493/logs/pull-11928-20190529-025555-9ba1a677-verify-fedora-30/log.html#80)

**Comment by KKoukiou** [comment]
> It looks like [this failure](https://209.132.184.41:8493/logs/pull-11928-20190529-034646-9ba1a677-verify-debian-testing/log.html#80) is related. It also happens on [fedora-30](https://209.132.184.41:8493/logs/pull-11928-20190529-025555-9ba1a677-verify-fedora-30/log.html#80)

Fixed.

**Comment by martinpitt** [comment]
[Some](https://209.132.184.41:8493/logs/pull-11928-20190529-074343-5e0035bd-selenium-firefox/log.html#50) [machine](https://209.132.184.41:8493/logs/pull-11928-20190529-074343-5e0035bd-verify-ubuntu-stable/log.html#77) flakes, retrying these.

**Comment by martinpitt** [comment]
The [same selenium/firefox](https://209.132.184.41:8493/logs/pull-11928-20190529-094040-5e0035bd-selenium-firefox/log.html) failure again..

**Comment by KKoukiou** [comment]
[testDelete](https://209.132.184.41:8493/logs/pull-11785-20190529-082525-7b2d0307-verify-ubuntu-stable/log.html#245) still fails from time to time, I nede to debug this further, :(

**Comment by martinpitt** [comment]
marking needswork for the selenium/firefox regression

**Comment by KKoukiou** [comment]
> The [same selenium/firefox](https://209.132.184.41:8493/logs/pull-11928-20190529-094040-5e0035bd-selenium-firefox/log.html) failure again..

These are not real failures, are are skips. I wonder why the test results show them like failures.

**Comment by KKoukiou** [comment]
> > The [same selenium/firefox](https://209.132.184.41:8493/logs/pull-11928-20190529-094040-5e0035bd-selenium-firefox/log.html) failure again..
> 
> ~These are not real failures, are are skips. I wonder why the test results show them like failures.~

Not sure who retried this PR, but this failure is not there anymore.
I looked at the test and it doesn't seem directly related to the PR either.
Rebasing and re-pushing to re-try all tests here.

**Comment by KKoukiou** [comment]
The selenium tests were fixed by something else which got merged, I rebased and reran tests two time, and no machines failures appeared. @martinpitt I guess this is ready to land /o

**Pull request merged**